config/crd/bases/network.schiff.telekom.de_layer2networkconfigurations.yaml

```diff
@@ -126,6 +126,7 @@ spec:
                 x-kubernetes-map-type: atomic
               vni:
                 description: VXLAN VNI Id for the layer 2 network
+                minimum: 1
```

Comment:
we should add the `maximum: 16777215` as well to also validate the upper boundary.